source/Glimpse.Site/App_Start/BundleConfig.cs

```diff
@@ -8,6 +8,9 @@ public class BundleConfig
         // For more information on bundling, visit http://go.microsoft.com/fwlink/?LinkId=301862
         public static void RegisterBundles(BundleCollection bundles)
         {
+            bundles.Add(new ScriptBundle("~/bundles/headsitejs").Include(
+                "~/Scripts/ApplicationInsightsSnippet.js"));
```

Comment:
We recommend to put it into the head to catch errors happening during pages and scripts parsing and have a chance to send telemetry if user leaving the page before the full loading. We can move it to the bottom of body tag, but I'd advice to keep it this way.